.env.example

```
@@ -0,0 +1,9 @@
+# Clerk Environment Variables
+# Get these from your Clerk dashboard: https://dashboard.clerk.com/
+VITE_CLERK_PUBLISHABLE_KEY=pk_test_XXXXXXXXXXXXXXXXXXXXXXXX
+
+# Your app's URL
+VITE_CLERK_SIGN_IN_URL=/login
+VITE_CLERK_SIGN_UP_URL=/signup
+VITE_CLERK_AFTER_SIGN_IN_URL=/app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_CLERK_AFTER_SIGN_IN_URL key should go before the VITE_CLERK_SIGN_IN_URL key

(UnorderedKey)